src/corvus.h

```diff
@@ -25,6 +25,13 @@
 #define CORVUS_ASKING -5
 #define CORVUS_READONLY -6
 
+#define RET_NOT_OK(expr)                   \
+    do {                                   \
+        int __r = expr;                    \
```

Comment:
Will [an other identifier be safer for this variable](https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier#answer-228797 "What are the rules about using an underscore in a C++ identifier?")?